data-node/src/main/java/org/graylog/datanode/process/FailuresCounter.java

```diff
@@ -47,4 +47,8 @@ public synchronized boolean failedTooManyTimes() {
     public synchronized void resetFailuresCounter() {
         this.counter = initialValue;
     }
+
+    public int getCounter() {
+        return counter;
```

Comment:
Its tiny, but I would name this variable `count`, not `counter`.

Reply:
agree, renamed